static/app/views/performance/transactionSummary/transactionTags/tagsHeatMap.tsx

```diff
@@ -185,7 +186,8 @@ const TagsHeatMap = (
         show: true,
         showMinLabel: true,
         showMaxLabel: true,
-        formatter: (value: number) => axisLabelFormatter(value, 'Count'),
+        formatter: (value: number) =>
+          axisLabelFormatter(value, aggregateOutputType('Count')),
```

Comment:
Out of curiosity, does this return `"number"`? It looks like it should be `"count()"`, but '"Count"' works because unknown values fall return `"number"`

Reply:
Yeah, that's exactly what's happening, i'm going to change aggregateOutputType('Count') to 'number', so the behaviour is the same.